src/Altinn.Platform/Altinn.Platform.Storage/DataCleanup/Services/BlobService.cs

```diff
@@ -60,13 +67,98 @@ await foreach (BlobItem item in container.GetBlobsAsync(BlobTraits.None, BlobSta
             catch (Exception e)
             {
                 _sasTokenProvider.InvalidateSasToken(instance.Org);
-                _logger.LogError(e, $"BlobSerivce // DeleteDataBlobs // Org: {instance.Org} // Exeption: {e.Message}");
+                _logger.LogError(e, $"BlobService // DeleteDataBlobs // Org: {instance.Org} // Exeption: {e.Message}");
+                return false;
+            }
+
+            return true;
+        }
+
+        /// <inheritdoc/>
+        public async Task<bool> DeleteInstanceBackup(string instanceOwnerPartyId, string instanceGuid)
+        {
+            BlobContainerClient container = await CreateBackupBlobClient();
+
+            try
+            {
+                await foreach (BlobItem item in container.GetBlobsAsync(BlobTraits.None, BlobStates.None, $"instances/{instanceOwnerPartyId}/{instanceGuid}", CancellationToken.None))
+                {
+                    container.DeleteBlobIfExists(item.Name, DeleteSnapshotsOption.IncludeSnapshots);
+                }
+            }
+            catch (Exception e)
+            {
+                _logger.LogError(e, "BlobService // DeleteInstanceBackup // Instance: {InstanceOwnerPartyId}/{InstanceGuid} // Exeption: {Exception}", instanceOwnerPartyId, instanceGuid, e);
```

Comment:
Can use string interpolation instead.

I'm not sure I like the idea of including the CallStack in the form of a custom string is the right thing. We already include the exception which should provide the actual StackTrace. 
Example:
```suggestion
                _logger.LogError(e, $"Exception when attempting to delete backup of Instance: {instanceOwnerPartyId}/{instanceGuid}");
```

Reply:
We add the mesages this way to be able to extract key information into seperate columns in the logs in AI